src/shared/clients/ecsClient.ts

```diff
@@ -77,6 +75,39 @@ export class DefaultEcsClient {
         }
     }
 
+    public async listTasks(cluster: string, serviceName: string): Promise<string[]> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.ListTasksRequest = { cluster: cluster, serviceName: serviceName }
+        try {
+            const listTasksResponse = await sdkClient.listTasks(params).promise()
+            return listTasksResponse.taskArns ?? []
+        } catch (error) {
+            getLogger().error(
+                `ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": ${error} `
+            )
+            throw error
+        }
+    }
+
+    public async describeTasks(cluster: string, tasks: string[]): Promise<ECS.Task[]> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.DescribeTasksRequest = { cluster, tasks }
+        try {
+            const describedTasks = await sdkClient.describeTasks(params).promise()
+            return describedTasks.tasks ?? []
+        } catch (error) {
+            getLogger().error(error)
+            throw error
+        }
+    }
+
+    public async describeServices(cluster: string, services: string[]): Promise<ECS.Service[]> {
```

Comment:
`listServices` is now calling `describeServices` too. So either revert the change and have `listServices` call `listServices` or remove this method and rename the other one to `describeServices`

Reply:
There are two API calls needed, `listServices` and `describeServices`.  To properly display the info we want, both calls are needed. We also need `describeServices` on it's own for other commands.  I have changed the name of any functions that utilize more than one API call as `get<Resource>` to try and show it's not just simply the return of a single API call.